NEWS.md

```diff
@@ -1,3 +1,5 @@
+* export of metadata to JSON now follows the convention NA -> null
```

Comment:
This makes it sound like it applies to all JSON output

Reply:
yes. will specify this then squash and merge